downloader/history_downloader/__init__.py

```diff
@@ -0,0 +1,12 @@
+import ccxt
+
+class HistoryDownloader:
+    exchange = ccxt.binance() #creating instance of binance exchange
+    def __init__(self,crypto_symbol:str)->None:

```

Comment:
Missing formating, remember to format your code.
```suggestion
    def __init__(self, crypto_symbol: str) -> None:

```